featuretools/feature_base/feature_base.py

```diff
@@ -421,8 +457,9 @@ class AggregationFeature(FeatureBase):
     # each time point during calculation
     use_previous = None
 
-    def __init__(self, base_features, parent_entity, primitive, use_previous=None,
-                 where=None):
+    def __init__(self, base_features, primitive,
```

Comment:
I made it optional because requiring it would force callers who only have a `relationship_path` to do `relationship_path[0].parent_entity`. In the constructor we then check that `parent_entity == relationship_path[0].parent_entity`, which in this case would always be True so its not adding any value.

But if we are more concerned with end users who are calling this directly then it makes sense to require it for extra safety.